packages/amplify-util-uibuilder/package.json

```diff
@@ -16,10 +16,10 @@
   "dependencies": {
     "@aws-amplify/amplify-cli-core": "4.0.1",
     "@aws-amplify/amplify-prompts": "2.6.6",
-    "@aws-amplify/codegen-ui": "2.11.1",
-    "@aws-amplify/codegen-ui-react": "2.11.1",
+    "@aws-amplify/codegen-ui": "2.12.0",
+    "@aws-amplify/codegen-ui-react": "2.12.0",
     "amplify-codegen": "^3.4.1",
-    "aws-sdk": "^2.1233.0",
+    "aws-sdk": "^2.1350.0",
```

Comment:
why do we need to bump aws-sdk?
if we want to do this we should make sure all packages in cli resolve same version.

Reply:
> why do we need to bump aws-sdk?
We need to bump the version to take advantage of the `StorageField` configurations we just added to the SDK. 
> if we want to do this we should make sure all packages in cli resolve same version.
I am seeing that the last time our team updated the SDK version, we did this by updating all the versions in the monorepo. I will rev with that change now.